src/components/ProjectCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styles from './ProjectCard.module.css';
+import { Props } from './types';
+
+function ProjectCard({
+    status,
+    client,
+    lead,
+    manager,
+    teamType,
+    startDate,
+    endDate,
+}: Props) {
+    let statusClass = null;
+
+    if (status === 'active') {
+        statusClass = styles.active;
+    } else if (status === 'inactive') {
+        statusClass = styles.inactive;
+    } else if (status === 'cancelled') {
+        statusClass = styles.cancelled;
+    }
```

Comment:
```suggestion
    statusClass = styles[status];
```

Reply:
Thanks for your comment! :) 
Changed as you suggested